From 76010d681a73a9f82e5e37b8246537ff0e2b2a10 Mon Sep 17 00:00:00 2001
From: Richard Hughes <richard@hughsie.com>
Date: Wed, 28 Aug 2024 23:36:04 +0100
Subject: [PATCH 06/10] Only support JCat for verification of metadata and
 firmware

This means we can adopt different checksums and signatures (PQ?) in the future
without making fwupd changes.

(cherry picked from commit 80d5f98eef9cc91724a4fbbcdab237cae4d1a722,
changes to contrib/qubes/ only)
---
 contrib/qubes/src/qubes_fwupdmgr.py | 7 -------
 1 file changed, 7 deletions(-)

diff --git a/contrib/qubes/src/qubes_fwupdmgr.py b/contrib/qubes/src/qubes_fwupdmgr.py
index 8fce44e63..646aa1a47 100755
--- a/contrib/qubes/src/qubes_fwupdmgr.py
+++ b/contrib/qubes/src/qubes_fwupdmgr.py
@@ -119,13 +119,6 @@ class QubesFwupdmgr(FwupdHeads, FwupdUpdate, FwupdReceiveUpdates):
             # actually needing refreshing
             if remote.get("Kind") != "download":
                 continue
-            # skip unsupported keyring kind
-            if remote.get("KeyringKind") not in ("jcat",):
-                print(
-                    "Skipping remote '{}' due to unsupported keyring type '{}'".format(
-                        name, remote.get("KeyringKind")
-                    )
-                )
             assert "MetadataUri" in remote
             remotes[name] = remote["MetadataUri"]
 
-- 
2.52.0

